src/clp_ffi_js/ir/UnstructuredIrStreamReader.cpp

```diff
@@ -144,7 +144,6 @@ auto UnstructuredIrStreamReader::decode_range(size_t begin_idx, size_t end_idx,
             };
         }
         message = parsed.value();
-        m_ts_pattern.insert_formatted_timestamp(log_event.get_timestamp(), message);
         return message;
```

Comment:
```suggestion
        return parsed.value();
```
Then we can delete the `std::string message;` declaration and the value assignment above.